modules/channel-vonage/src/backend/client.ts

```diff
@@ -113,6 +119,7 @@ export class VonageClient {
         payload = (await this.handleOptionResponse(text, userId, conversation.id)) ?? payload
         break
       case 'audio':
+        // TODO: Link received from Nexmo/Vonage API are only valid for 10min
```

Comment:
So we'll need to get file and store it in media storage if we want to persist it ? do you think there's a use case for it ?

Reply:
See this `TODO` more as a reminder for a close future where we'll need to add support for image/audio/video reception.

Since the user might not know that the link we passed is short-lived, I think we need to do something so the user avoids storing or reusing this link.

Here are the two choices I have in mind right now:

- We can store the file in the media storage and then forward the link to the bot to be handled.
- Or we can "force" the developer to handle file reception in a short amount of time and do something with the file (not the best idea imho)